src/components/tx/TxData.js

```diff
@@ -466,22 +467,24 @@ class TxData extends React.Component {
       if (hashes.length === 0) {
         return null;
       }
-      if (hashes.length === 1) {
-        const h = hashes[0];
+      const ncFailedExecution = 'Nano contract failed execution';
+      const renderSingleElement = h => {
+        if (h === NANO_CONTRACT_EXECUTION_FAIL) {
+          return <span>{ncFailedExecution}</span>;
```

Comment:
suggestion: You could inline `ncFailedExecution` here, as it's the only place it's used..